hooks/useUTMs.ts

```diff
@@ -83,9 +83,10 @@ const useUTMs = () => {
 
   /**
    * A function that returns a boolean if the passed in string is an absolute URL and is under the stanford.edu domain.
+   * Handles multiple subdomain levels like www.gsb.stanford.edu
    */
   const isStanfordUrl = (url: string) => {
-    const stanfordRegex = new RegExp('^(https?:)?(\/\/)?([a-zA-Z0-9-]+\.)?(stanford\.edu).*');
+    const stanfordRegex = /^(https?:)?(\/\/)?([a-zA-Z0-9.-]*\.)?stanford\.edu/;
```

Comment:
Update this stanford domain check to include multiple subdomain levels - there was a link in the mega menu that  www.gsb.stanford.edu that didn't pass the check before this update. It's forwarded to the non www gsb.stanford.edu anyway so technically we could just update the link but I don't know what else is out there 😂